decidim-core/app/mailers/decidim/user_update_mailer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Decidim
+  class UserUpdateMailer < ApplicationMailer
+    def notify(user, updates)
+      with_user(user) do
+        @user = user
+        @organization = user.organization
+        @updates = format_array(updates)
+        mail(to: user.email, subject: I18n.t(
+          "decidim.user_update_mailer.subject"
+        ))
+      end
+    end
+
+    private
+
+    def format_array(updates)
+      case updates.length
+      when 1
+        updates.first
+      when 2
+        "#{updates.first} and #{updates.last}"
+      else
+        "#{updates[0..-2].join(", ")}, and #{updates.last}"
+      end
+    end
```

Comment:
As i understand it, this will get the list of the updated fields, and it would create a string like `field1, field2, and field3`

This is not ok, as this string is not translatable, generating strings like : Nickname, Despre, *and* Url Personal 
Please mind the "and".

Reply:
Actually, those fields are translated attributes, but yes, thanks for noticing this translation issue